README.md

```diff
@@ -7,94 +7,94 @@
-- Support for animated textures ( like Aerith waterfall, light fading, etc. )
-- Support for soft-reset while you're playing, like on the PSX!
-- Support for battle toggle ( enable/disable at your own pleasure )
-- **Voice acting**! One of the first mods to make use of this will be [Echo-S](https://forum.tsunamods.com/viewtopic.php?f=65&t=9) when it will be released!
+- 60 FPS
+- eStore support! No game converter required
+- Vertical centering for fields and movies
+- Fullscreen battle scenes
+- Menu cursor vertically aligned on the center of words
+- Movies continue to play while running in an inactive window
+- Movie volume respects global sound volume
+- Steam sound and music volume configuration preservation (configure at your pleasure and on the next run it will be inherited)
+- Configurable background transparency in battle dialogs (by default set to 75%)
+- SFX volume change applies in real-time, instead of requiring a game reload
+- Support for animated textures (like Aerith's waterfall, light fading, etc.)
+- Support for soft-reset while you're playing, just like the PSX
+- Support for battle toggle (enable/disable at your own pleasure)
+- **Voice acting**! [Echo-S](https://forum.tsunamods.com/viewtopic.php?f=65&t=9) was the first mod to take advantage of this!
 - Support for external SFX audio effects
-- Support for external Ambient audio effects
-- Support for external movie audio files ( allows multiple videos to share the same audio )
-- Support for external movie voice acting ( dedicated audio layer only for voice acting on top of movies )
-- Steam achievements! You can now unlock them while playing with FFNx
-- Real-time light engine! You can now feel the game visually like never before
+- Support for external ambient audio effects
+- Support for external movie audio files (allows multiple videos to share the same audio)
+- Support for external movie voice acting (dedicated audio layer only for voice acting on top of movies)
+- Steam achievements can be unlocked while playing within FFNx
+- Real-time light engine - You can now feel the game visually like never before
 - Real-time camera control in battles
 - Analogue controls using the full axis of your left analog stick
 
 #### FF8
-- Vibration support!
-- **Voice acting**!
-- Maki's patch for Worldmap included
+- Vibration support
+- **Voice acting**
+- Maki's patch for worldmap included
 - Enable the VRAM debug window while playing in order to see how the engine uploads textures
 
 ### As a modder
 
-- Game rendering inspection through [RenderDoc](https://renderdoc.org/)!
-- [DDS Texture support](https://beyondskyrim.org/tutorials/the-dds-texture-format) up to BC7 format, with PNG Texture support as fallback.
+- Game rendering inspection through [RenderDoc](https://renderdoc.org/)
+- [DDS Texture support](https://beyondskyrim.org/tutorials/the-dds-texture-format) up to BC7 format, with PNG support as fallback
```

Comment:
Can you please update also the URL to https://wiki.beyondskyrim.org/wiki/Arcane_University:DDS_Data_Format ?